### ISSUE TITLE
The client does not receive a message after network recovery

### ISSUE THREAD

**Issue by AnnePersonal** [cometd/cometd#715]
In the use of Cometd process, if the handshake users to reach 1000 sessions, each session push message about 200, there will be a loss of information, a small amount does not exist, there is broken network, the session did not remove, then access the Internet, off Network message, will not be sent to the session again?

**Comment by sbordet** [comment]
I'm sorry but I cannot decipher what you wrote.

Can you please state clearly what are you doing in details (including configuration), what is the expected result, and what you get instead ?

**Comment by AnnePersonal** [comment]
I used Message Acknowledgment Extension in server and client, However, the client receives more messages than the server. I need to set other properties or Extension ？

**Comment by sbordet** [comment]
I'm sorry again, but I cannot really help you. What does "the client receives more messages than the server" mean ? That could be perfectly normal.

You have to detail exactly what are you doing, what is the expected result, and what you get instead.
Attach a test case that shows the issue. Most likely is not a CometD problem.

Also, be sure you have read the [documentation](https://docs.cometd.org/).

**Comment by AnnePersonal** [comment]
Hello sbordet,

Thanks for your patience, any help would be appreciated. I came accross the following scenario when I am using cometD 3.0

There are 1000 clients subscribed one chanel by handshake with Message Acknowledgement Extension to make sure message publish successful. And then server send 100 piece messages, but I found all the clients received about 1008000 piece messages in total. With my understanding it should be 100000 piece messages in total. 

Any idea or suggestion would be help. Thanks.
